packages/frontend/src/components/shifts/ShiftBlock.tsx

```diff
@@ -134,6 +153,19 @@ function ShiftBlock(props: RootProps) {
           />
         )}
       </StyledShiftBlock>
+      <Popper
+        id={id}
+        open={popperOpen}
+        anchorEl={anchorEl}
+        onMouseEnter={handlePopperClose}
+      >
+        <Box sx={{ border: 1, p: 1, bgcolor: 'background.paper' }}>
+          {shiftViewModel &&
+            `${shiftViewModel.scheduleName} -- ${TimeOfDayFormatter.format(
+              shiftViewModel.shift.startTime,
+            )} to ${TimeOfDayFormatter.format(shiftViewModel.shift.endTime)} `}
```

Comment:
Nit but I find inline conditionals in JSX kind of confusing... prefer to see the control flow outside of JSX, but not a big deal and def a matter of preference

Reply:
i like having a standard. we can go with that. thanks for the rec